src/components/ui/Input.js

```diff
@@ -0,0 +1,71 @@
+import styled from 'styled-components'
+
+const Input = ({ placeholder, onChange, onEnter, label, icon, className }) => {
+  const id = label?.toLowerCase().split(' ').join('-').concat(`-${className}`)
```

Comment:
Are `name` and `id` necessary for the `input`? It doesn't seem like it gets used anywhere (no selection by id, and the input isn't being used by a form).

Reply:
Potentially you may want to spread remaining props into the HTML input, so that the user can pass their own id and name as they wish

Reply:
Its an A11y feature. Name is only defined when there is a label so that when a user clicks said label the focus is put onto the relevant `<input />` element on the page. 

I will remove the `id` attribute though.

Reply:
Oh gotcha, yes this is important due to the label!